tests/integration/components/events-test.js

```diff
@@ -0,0 +1,21 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | events', function (hooks) {
+  setupRenderingTest(hooks);
+
+  // TODO: add tests after events section is out of feature flag
```

Comment:
Why not testing now?

Reply:
Event component rendered empty before because of feature flag and how I was inserting the component. Since this is now changed, have added the tests for event component.